core/templates/dev/head/app.js

```diff
@@ -1220,3 +1220,20 @@ oppia.factory('codeNormalizationService', [function() {
     }
   };
 }]);
+
+// Service for retrieving algorithm id of classifier for a given interaction id.
+oppia.factory('algorithmIdRetrievalService', [function() {
```

Comment:
Is there a case where we just want the algorithmId, or can we just return the PredictionService?

Also, I'm a bit confused about the use of the words algorithm in some places and predictionservice in others. Is there a difference? If they're being used as synonyms, we should just stick to one.